# Goal
float64 duration_sec
---
# Result
bool success
string message
---
# Feedback
float32 progress_percent